src/main/tray/tray.ts

```diff
@@ -3,43 +3,86 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 
-import path from "path";
 import packageInfo from "../../../package.json";
-import { Menu, Tray } from "electron";
+import { Menu, NativeImage, nativeImage, nativeTheme, Tray } from "electron";
 import { autorun, IComputedValue } from "mobx";
 import { showAbout } from "../menu/menu";
 import { checkForUpdates, isAutoUpdateEnabled } from "../app-updater";
 import type { WindowManager } from "../window-manager";
 import logger from "../logger";
-import { isDevelopment, isWindows, productName } from "../../common/vars";
+import { isWindows, productName } from "../../common/vars";
 import { exitApp } from "../exit-app";
-import { toJS } from "../../common/utils";
+import { getOrInsertWithAsync, toJS } from "../../common/utils";
 import type { TrayMenuRegistration } from "./tray-menu-registration";
+import parseDataURL from "data-urls";
```

Comment:
Do we really need one more dependency?
If you're not going to use `.body` from this package (which is not the case in that PR) it would just enough to use this for getting UTF-8 string:

```ts
utils/base64.decode(url.split("base64,")[1])
```

Reply:
Good point will remove dependency.